pkg/postage/postagecontract/contract.go

```diff
@@ -263,49 +263,49 @@ func (c *postageContract) getBalance(ctx context.Context) (*big.Int, error) {
 	return abi.ConvertType(results[0], new(big.Int)).(*big.Int), nil
 }
 
-func (c *postageContract) CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) ([]byte, error) {
+func (c *postageContract) CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) (string, []byte, error) {
 
 	if depth <= BucketDepth {
-		return nil, ErrInvalidDepth
+		return "", nil, ErrInvalidDepth
```

Comment:
i would use naked returns or named returns values to make the change more "pretty" and get rid of all of the quotes